user_manual/files/access_webgui.rst

```diff
@@ -283,18 +283,79 @@ change     Allows uploading a new version of a shared file and replacing it
 delete     Allows the users you share with to delete shared files
 ========== ===================================================================
    
+Custom Groups
+-------------
+
+In previous versions of ownCloud, if you wanted to share a file or a folder with more than one person, you had to share it  either with many people individually, or share to one or more groups. 
+However, you could only share with groups which your ownCloud administrator had already created. 
+
+This wasn't the most efficient way to work. 
+To address that, as of ownCloud 10.0, you can now create your own groups on-the-fly, through a feature called "Custom Groups". 
+Here's how to use it. 
+
+Creating Custom Groups
+~~~~~~~~~~~~~~~~~~~~~~
+
+Assuming that your ownCloud administrator's already `enabled custom groups`_; under the admin menu, in the top right-hand corner, click "**Settings**" (1).
+Then, in the main menu on the settings page, in "**Personal**" section, click the option: "**Customgroups**" (2).
+This will take you to the "**Custom Groups**" admin page. 
+
+.. image:: ../images/custom-groups/owncloud-create-custom-group-annotated.png
+   :alt: The Custom Groups administration panel
+
+To create a new custom group, in the text field at the top where you see the placeholder text: "**Group name**", add the group name and click "**Create group**".
+After a moment or two, you’ll see the new custom group appear in the groups list.
+
+.. note::
+   Please be aware that custom groups are only visible to every member of the group, but not to anyone else outside the group.
+
+Managing Group Members
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/custom-groups/custom-group-manage-group-members.png
+   :alt: Manage members in a custom group
+
+To add or remove users in a custom group, click your role (1), which will likely be "**Member**" (at least at first), and you'll see a panel appear on the right-hand side listing the group's users and their roles. 
+In the "**Add user to this group field**" at the top of the panel (2), start typing the name of the user that you want to add. 
+
+After a moment or two, you'll see a list of users that match what you've typed appear (if there are any) in a popup list. 
+Click the one that you want, and they'll be added to the group. 
+Finally, you’ll see a confirmation at the top of the page (3), indicating that the user’s been added to the custom group.
+
```

Comment:
arr, yes, looks like there's something wrong here. thanks for discovering/reporting! Actually oC admins are 'group admins' for all custom groups afaik. @PVince81 can you clarify?